Packs/SymantecMSS/Integrations/SymantecMSS/SymantecMSS.py

```diff
@@ -86,7 +87,7 @@ def api_call(body, headers):
         res = requests.post(url=SERVER_URL + "/SWS/incidents.asmx", cert=cert, data=body, headers=headers)
         if res.status_code < 200 or res.status_code >= 300:
             raise Exception(
-                "Got status code " + str(res.status_code) + " with body " + res.content + " with headers " + str(
+                "Got status code " + str(res.status_code) + " with body " + str(res.content) + " with headers " + str(
```

Comment:
why this change?

Reply:
res.content can also be a bytes object so ruff or mypy were complaining.